config/checkstyle-examples-suppressions.xml

```diff
@@ -55,6 +55,14 @@
   <suppress id="discouragedRegexpInViolationMessage"
     files="operatorwrap[\\/]Example2.java"/>
 
+  <!-- it prints user regexp -->
+  <suppress id="discouragedRegexpInViolationMessage"
+            files="regexponfilename[\\/]Example[24].*"/>
```

Comment:
Please remove unrelated updates.
We expect here only change about `avoidstarimport`